ztm-front/src/types/VoidFunction.ts

```diff
@@ -0,0 +1 @@
+export type VoidFunction<T> = (arg0: T) => void
```

Comment:
I am not sure about that, usually, if we pass any param we write it `onClick(arg: string): void`
in this case, when you will have the second argument, this function is not useful
please ask Jacek for it

Reply:
+1. User Voide function as void function, other function should be typed inline